xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h

```diff
@@ -576,6 +576,37 @@ extern "C"
    */
   bool CanSeekStream();
 
+  /*
+   * Check if the given EPG tag can be recorded.
+   * @param tag the epg tag
+   * @param isRecordable Set to true if the tag can be recorded
+   * @return PVR_ERROR_NO_ERROR if isRecordable was set by the addon
+   * @remarks Optional, return PVR_ERROR_NOT_IMPLEMENTED to let kodi decide
+   *
+   */
+  PVR_ERROR IsRecordable(const EPG_TAG &tag, bool *isRecordable);
+
+  /*
+   * Check if the given EPG tag can be played.
+   * @param tag The EPG tag
+   * @return True if the EPG tag can be played
+   * @remarks Required, always return false if not supported by the addon
+   *
+   */
+  bool IsPlayable(const EPG_TAG &tag);
```

Comment:
Please see the comments I accidentally made for CPVRClient your changes and apply them here. ;-)